client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -560,8 +744,8 @@ public Schema lookUpSubjectVersion(Map<String, String> requestProperties,
     String path = builder.build(subject).toString();
 
     Schema schema = httpRequest(path, "POST",
-                                registerSchemaRequest.toJson().getBytes(StandardCharsets.UTF_8),
-                                requestProperties, SUBJECT_SCHEMA_VERSION_RESPONSE_TYPE_REFERENCE);
+        registerSchemaRequest.toJson().getBytes(StandardCharsets.UTF_8),
```

Comment:
It seems a lot of changes in this file are reformatting.  Can we keep the old formatting?  This will make it easier in case we ever want to cherry-pick this PR.